### PR TITLE
immudb 0.7.1

### PR DESCRIPTION
**Pull request by jeroiraz**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/immudb.rb [hunk 1]
```diff
@@ -21,8 +21,16 @@ def install
 
   test do
     immudb_port = free_port
+
+    ENV["IMMUDB_AUTH"] = "false"
+    ENV["IMMUCLIENT_AUTH"] = "false"
+
+    tmp_data_dir = "/var/tmp/immudb"
+
+    rm_rf(tmp_data_dir)
```
Comment (SMillerDev) [inline Formula/immudb.rb:30]: Please just make a directory in the current location, test directories only live temporarily so it'll be automatically removed.

Comment (jeroiraz) [reply Formula/immudb.rb]: thanks @SMillerDev, already changed

File: Formula/immudb.rb [hunk 2]
```diff
@@ -42,6 +49,10 @@ def install
     response = shell_output(command.join(" "))
     assert_equal "world", Base64.decode64(JSON.parse(response)["value"])
 
-    assert_match "Uptime", shell_output("#{bin}/immuadmin stats -t -p #{immudb_port}")
+    shell_output("#{bin}/immuadmin stats -t -p #{immudb_port}")
+
+    pid = File.read(pidfile)
+
+    Process.kill("KILL", pid.to_i)
```
Comment (SMillerDev) [inline Formula/immudb.rb:56]: All test processes will be killed by brew after the test, no need to do it manually

Comment (jeroiraz) [reply Formula/immudb.rb]: Thanks, changed.

**Comment by dawidd6** [comment]
Cancelled workflow run as it definitely lasts too long. Something's wrong with the test I think, as it hangs on this step:
```
==> brew test --verbose immudb
##[error]The operation was canceled.
```

**Comment by jeroiraz** [comment]
thanks @dawidd6. When building the formula from source test passes on macOS Catalina, but as you pointed out, when the bottle for Catalina is being tested, it never ends. Any hint?

**Comment by dawidd6** [comment]
Spotted the same yesterday, I don't really know what causes that. Weird.

**Comment by SMillerDev** [comment]
Does the test stop correctly for you locally?

**Comment by jeroiraz** [comment]
@SMillerDev, yes it does without any issue. The steps I locally run are:
brew uninstall immudb
brew install --build-from-source immudb
brew test immudb

**Comment by SMillerDev** [comment]
It works 🥳, can you squash your commits please?

**Review by SMillerDev — APPROVED** [review]
Thanks @jeroiraz  ! Without contributions like yours it'd be impossible to keep homebrew going with the high standards that users have come to expect from the project. You can feel good knowing that you've made the world a tiny bit better for homebrew users around the world! 👍 🎉

**Pull request closed without merging**